From f90d4d5548dfe2d75e83b8da91af9be0e261286b Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Sun, 30 May 2021 18:00:26 +0200
Subject: [PATCH] Fix kcmfontinst install destination

This reverts part of a8ec95411a9f6, KCModuleLoader can't load kcm_fontinst
if the library is installed in KDE_INSTALL_PLUGINDIR/kcms/; looks like that
only works for KCMs using QML.

BUG: 436306
FIXED-IN: 5.22.1
(cherry picked from commit 81ae7f8e28389edcd08e247498c5ee9ad4977f83)
---
 kcms/kfontinst/kcmfontinst/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/kcms/kfontinst/kcmfontinst/CMakeLists.txt b/kcms/kfontinst/kcmfontinst/CMakeLists.txt
index fca680926..99dbbd311 100644
--- kcms/kfontinst/kcmfontinst/CMakeLists.txt
+++ kcms/kfontinst/kcmfontinst/CMakeLists.txt
@@ -18,8 +18,7 @@ target_link_libraries(kcm_fontinst
     X11::X11
 )
 
-set_target_properties(kcm_fontinst PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin/kcms")
-install(TARGETS kcm_fontinst  DESTINATION ${KDE_INSTALL_PLUGINDIR}/kcms)
+install(TARGETS kcm_fontinst  DESTINATION ${KDE_INSTALL_PLUGINDIR})
 install( FILES fontinst.desktop  DESTINATION  ${KDE_INSTALL_KSERVICES5DIR} )
 install( FILES kfontinst.knsrc DESTINATION  ${KDE_INSTALL_KNSRCDIR} )
 
-- 
GitLab

